main.go

```diff
@@ -219,6 +231,41 @@ func DoStashCPSingle(sourceFile string, destination string, methods []string, re
 		}
 	}()
 
+	exporter, err := otlptrace.New(
+		context.Background(),
+		otlptracehttp.NewClient(otlptracehttp.WithEndpoint("osdf-oltp.nrp-nautilus.io:443")),
```

Comment:
Can we register a `osg-htc.org` hostname for this?  I'm wary of leaking implementation hostnames into source code.

Reply:
Yup, this is very much a proof-of-concept pull request.  The plan is to move it over to tiger, but yes, we will have an osg-htc hostname.